core/node/node_framework/src/implementations/layers/state_keeper/mod.rs

```diff
@@ -32,12 +33,22 @@ use crate::{
 ///
 #[derive(Debug)]
 pub struct StateKeeperLayer {
-    db_config: DBConfig,
+    state_keeper_db_path: String,
+    state_keeper_db_block_cache_capacity: usize,
+    state_keeper_db_max_open_files: Option<NonZero<u32>>,
```

Comment:
Bikeshedding: IMO, for forward compatibility it would be beneficial to use `RocksdbStorageOptions` here instead and accept it as an arg for `StateKeeperLayer::new()`.